package.json

```diff
@@ -10,7 +10,7 @@
     "postinstall": "husky",
     "lint": "npm run lint:js && npm run lint:css && npm run lint:md",
     "lint:css": "stylelint 'src/**/*.{js,ts,tsx}'",
-    "lint:js": "eslint ./ --max-warnings 0",
+    "lint:js": "eslint ./",
```

Comment:
At this point, they are still 716 deprecation warnings that will be addressed in future PRs. Temporarily removing the `--max-warning` flag for now.

Reply:
That feel like a good trade-off – keeps em in our face